src/components/Table/index.tsx

```diff
@@ -60,6 +65,25 @@ const Table: React.FunctionComponent<Props<any>> = ({
 		},
 	}
 
+	React.useEffect(() => {
+		const tableBody = document.querySelector('.ant-table-body')
```

Comment:
1 problem with this is `querySelector` only matches the first element: 
https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector#Finding_the_first_element_matching_a_class
So when there are 2 tables in a view and we want pagination on both of them, this will not work with the second one. Besides, we are dealing with a reusable table.
I personally think this pagination logics should be done in Table's parent component. Some table will not need it at all so it sounds a bit of resource-wasting to subscribe for scroll event on those who do not need pagination.
Besides, we are using with React so the best way to select a DOM element is using `ref`. Even though I am not sure how we should do it on 3rd party lib like Ant.

Reply:
The sure thing is that using ref would be better. However, I haven't found a way to use it yet. We could use id to avoid querySelector issue